src/main/io/debugserial.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "platform.h"
+
+#define DBG_DEFAULT_BAUDRATE 115200
+#define DBG_DEFAULT_PORT SERIAL_PORT_USART3
+
+#ifdef USE_DEBUG_SERIAL
+	#define __DBG_SETRATE(x)    dbg_setRate(x)
```

Comment:
double underscore (`__`) is  a bit overused here(it is reserved in C).

Also with optimization and inlining, it should be enough to use `if(USE_DEBUG_SERIAL)` and avoid macros.

Or you can use something like ```
static inline void dbg_nop(void) { }
#ifdef USE_DEBUG_SERIAL
# define DEBUG_ENABLED(x) x
#else
# define DEBUG_ENABLED(x) dbg_nop()
#endif
#define __DBG_TICK()		DEBUG_ENABLED(dbg_tick())